features/admin.extensions.v1/configs/endpoints.ts

```diff
@@ -42,12 +42,14 @@ export const getExtendedFeatureResourceEndpoints = (serverHost: string,
         inviteEndpoint: `${ serverHost }/api/asgardeo-guest/v1/users/invite`,
         inviteLinkEndpoint: "/api/users/v1/offline-invite-link",
         notificationSendersEndPoint: `${ serverHost }/api/server/v1/notification-senders`,
+        onPremUserStoreAgentConnection: `${ serverHost }/api/onprem-userstore/v1/connection`,
```

Comment:
on prem user store doesn't need an agent to connect, does it?

```suggestion
        onPremUserStoreConnection: `${ serverHost }/api/onprem-userstore/v1/connection`,
```

Reply:
This is the existing user store for cloud offering, which connects through an agent.